tools/openshift-ci/verification_tests_ci_entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+[[ -z "$BUSHSLICER_TEST_ENVIRONMENT" ]] && { echo "BUSHSLICER_TEST_ENVIRONMENT not set"; exit 1; }
+[[ -z "$BUSHSLICER_TEST_CLUSTER" ]] && { echo "BUSHSLICER_TEST_CLUSTER not set"; exit 1; }
+[[ -z "$BUSHSLICER_TEST_TOKEN" ]] && { echo "BUSHSLICER_TEST_TOKEN not set"; exit 1; }
+[[ -z "$BUSHSLICER_TEST_CONFIG" ]] && { echo "BUSHSLICER_TEST_CONFIG not set"; exit 1; }
+[[ -z "$BUSHSLICER_TEST_FORMAT" ]] && { echo "BUSHSLICER_TEST_FORMAT not set"; exit 1; }
+if [ -z "$BUSHSLICER_TEST_RESULTS" ]
+then
+      echo "BUSHSLICER_TEST_RESULTS not set, setting to current dir"
+      export BUSHSLICER_TEST_RESULTS="$PWD/junit-report"
+fi
+
+export BUSHSLICER_DEFAULT_ENVIRONMENT="$BUSHSLICER_TEST_ENVIRONMENT"
+[[ -z "$BUSHSLICER_DEFAULT_ENVIRONMENT" ]] && { echo "BUSHSLICER_DEFAULT_ENVIRONMENT not set"; exit 1; }
+
+export OPENSHIFT_ENV_$(echo "$BUSHSLICER_TEST_ENVIRONMENT" | awk '{print toupper($0)}')_HOSTS="${BUSHSLICER_TEST_CLUSTER}:etcd:master:node"
+export OPENSHIFT_ENV_$(echo "$BUSHSLICER_TEST_ENVIRONMENT" | awk '{print toupper($0)}')_USER_MANAGER_USERS=:"${BUSHSLICER_TEST_TOKEN}"
+export OPENSHIFT_ENV_$(echo "$BUSHSLICER_TEST_ENVIRONMENT" | awk '{print toupper($0)}')_WEB_CONSOLE_URL=https://${BUSHSLICER_TEST_CLUSTER}/console
+
+export TESTENV="BUSHSLICER_V3"
+export BUSHSLICER_CONFIG="${BUSHSLICER_TEST_CONFIG}"
```

Comment:
This is where we would put in our config overrides, for example, marionette driver, etc. This is being supplied higher up in the ci operator, which is why it's being set as an ENV var here.

Reply:
why don't we rename variable in template instead of copying variables here? In any case it can live forthe time being, but I fear nobody goes back to fix.